src/components/NewCommentForm.tsx

```diff
@@ -45,24 +122,31 @@ export const NewCommentForm: React.FC = () => {
             name="email"
             id="comment-author-email"
             placeholder="email@test.com"
-            className="input is-danger"
+            className={classNames('input', { 'is-danger': checkEmailName })}
+            value={formInputs.email}
+            onChange={(e) => handleChange(e)}
```

Comment:
```suggestion
            onChange={handleChange}
```
you don't need wrapper function in such cases